app/src/main/java/com/example/habitsmasher/MainActivity.java

```diff
@@ -25,30 +28,30 @@
  */
 public class MainActivity extends AppCompatActivity {
 
-    private ActivityMainBinding binding;
-    AppBarConfiguration appBarConfiguration;
+    private ActivityMainBinding _binding;
+    AppBarConfiguration _appBarConfiguration;
+    private NavController _navController;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        binding = ActivityMainBinding.inflate(getLayoutInflater());
-        setContentView(binding.getRoot());
+        _binding = ActivityMainBinding.inflate(getLayoutInflater());
+        setContentView(_binding.getRoot());
 
         BottomNavigationView navView = findViewById(R.id.nav_view);
         // Passing each menu ID as a set of Ids because each
         // menu should be considered as top level destinations.
-        appBarConfiguration = new AppBarConfiguration.Builder(
+        _appBarConfiguration = new AppBarConfiguration.Builder(
                 R.id.navigation_home, R.id.navigation_dashboard, R.id.navigation_notifications, R.id.user_login)
                 .build();
 
         NavHostFragment navHostFragment = (NavHostFragment) getSupportFragmentManager().findFragmentById(R.id.nav_host_fragment_activity_main);
-        NavController navController;
         if (navHostFragment != null) {
-            navController = navHostFragment.getNavController();
-            NavigationUI.setupActionBarWithNavController(this, navController, appBarConfiguration);
-            NavigationUI.setupWithNavController(binding.navView, navController);
-            navController.navigate(R.id.action_navigation_home_to_LoginFragment);
+            _navController = navHostFragment.getNavController();
+            NavigationUI.setupActionBarWithNavController(this, _navController, _appBarConfiguration);
+            NavigationUI.setupWithNavController(_binding.navView, _navController);
+            _navController.navigate(R.id.user_login);
```

Comment:
On launch, boot up the login screen. Before it was actually booting up the home screen and triggering an automatic action to the login screen.